### PR TITLE
RDAPI-23379 Add known issues to API Gateway release notes May 22

### PR DESCRIPTION
**Pull request by dshielaxway**
Document known issues when multiple API Manager traffic ports are configured

Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Document known issues when multiple API Manager traffic ports are configured

## Deploy preview link

https://deploy-preview-2277--axway-open-docs.netlify.app/docs/apim_relnotes/20220530_apimgr_relnotes/#known-issues

_Although Netlify will add this link below, it will redirect to the main page of the documentation, not to an specific page._

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ ] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/apim_relnotes/20220530_apimgr_relnotes.md [hunk 1]
```diff
@@ -123,6 +123,44 @@ A Policy Studio performance issue has been observed when utilizing the YAML enti
 
 Related Issue: RDAPI-26745
 
+### API Catalog Swagger 2 export issue when multiple API Manager traffic ports configured
```
Comment (cgardineraxway) [inline content/en/docs/apim_relnotes/20220530_apimgr_relnotes.md:126]: Other parts of documentation refer to Swagger 2 as **Swagger 2.0** in all cases in which it is referenced.

**Review by credmond-axway — APPROVED** [review]

File: content/en/docs/apim_relnotes/20220530_apimgr_relnotes.md (hunk 1 above)
Comment (dshielaxway) [reply content/en/docs/apim_relnotes/20220530_apimgr_relnotes.md]: Thanks Cian. Updated.

**Review by credmond-axway — APPROVED** [review]

File: content/en/docs/apim_relnotes/20220530_apimgr_relnotes.md [hunk 2]
```diff
@@ -123,6 +123,44 @@ A Policy Studio performance issue has been observed when utilizing the YAML enti
 
 Related Issue: RDAPI-26745
 
+### API Catalog Swagger 2.0 export issue when multiple API Manager traffic ports configured
+
+If multiple traffic ports are defined when a Swagger 2.0 export from the API Manager Catalog is performed, and an attempt is made to subsequently reimport that generated document into another API Manager, multiple Backend URLs (one for HTTP and one for HTTPS) will be correctly generated, but the port will be incorrect for one of URLs. This is due to an inherent flaw in Swagger 2.0 as it only permits one host to be specified. At export-time API Manager takes the first SSL-based basePath, if one exists, from the list of available basePaths (host:port) and applies that to the `$.host` field in the resultant Swagger 2.0 definition.
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20220530_apimgr_relnotes.md:128]: @dshielaxway  @cgardineraxway I'm trying to simplify this paragraph a bit, and change verb tense to present as much as possible. I came up with this, please let me know if it's tech accurate. Thanks

When exporting Swagger 2.0 from API Manager Catalog with multiple traffic ports defined, if you try to subsequently reimport the generated document into another API Manager, the back-end URLs (HTTP and HTTPS) are correctly generated but the port will be incorrect for one of the URLs.

The issue is caused by an inherent flaw in Swagger 2.0 as it only permits one host to be specified. At export time, API Manager takes the first SSL-based basePath, if one exists, from the list of available basePaths (host:port) and applies that to the `$.host` field in the resultant Swagger 2.0 definition.

Comment (dshielaxway) [reply content/en/docs/apim_relnotes/20220530_apimgr_relnotes.md:128]: @andreamussap That looks good to me. Thanks!

**Pull request merged**